--- libircclient-1.6/configure.in~	2018-09-25 00:24:26.957726909 +0200
+++ libircclient-1.6/configure.in	2018-09-25 00:30:27.513160280 +0200
@@ -101,7 +101,7 @@
 
 if test "$use_openssl" = "yes"; then
 	AC_CHECK_LIB(crypto, [CRYPTO_new_ex_data], [], [AC_MSG_ERROR([OpenSSL not found])])
-	AC_CHECK_LIB(ssl,    [SSL_library_init], [], [AC_MSG_ERROR([OpenSSL not found])])
+	AC_CHECK_LIB(ssl,    [SSL_CTX_new], [], [AC_MSG_ERROR([OpenSSL not found])])
 	AC_CHECK_HEADER([openssl/ssl.h], [], [AC_MSG_ERROR([OpenSSL headers not found; did you install the -dev package?])])
 	CFLAGS="$CFLAGS -DENABLE_SSL"
 fi
